### DISCUSSION TITLE
Is Gradient Checkpointing Available For MACE?

### DISCUSSION THREAD

**Discussion by rydeveraumn** [ACEsuit/mace#1166] [Q&A]
Hello all,

I am trying to increase my effective batch size and generally I enable gradient checkpointing. Is that feature available for MACE models?

Warm regards,

Ryan

**Comment by ilyes319** [comment]
Hey! It is not implemented atm but happy to merge a PR if you ever do implement it.

**Reply by rydeveraumn** [reply]
I gave it a valiant effort today (6 hours). I don't know if it is my version of `pytorch` (2.2.0) but it seems to not work with torch scripting.
